collect_app/src/main/java/org/odk/collect/android/views/TrackingTouchSlider.kt

```diff
@@ -0,0 +1,80 @@
+package org.odk.collect.android.views
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.content.res.ColorStateList
+import android.util.AttributeSet
+import android.view.MotionEvent
+import android.view.View
+import com.google.android.material.R
+import com.google.android.material.color.MaterialColors
+import com.google.android.material.slider.Slider
+
+@SuppressLint("ClickableViewAccessibility")
+class TrackingTouchSlider(
+    context: Context,
+    attrs: AttributeSet?
+) : Slider(context, attrs), Slider.OnSliderTouchListener {
+    private var listener: OnChangeListener? = null
+
+    private var defaultTickActiveTintList: ColorStateList = tickActiveTintList
+    private var defaultThumbWidth = thumbWidth
+    private var defaultThumbTrackGapSize = thumbTrackGapSize
+
+    var isTrackingTouch: Boolean = false
+        private set
+
+    init {
+        addOnSliderTouchListener(this)
+        setLabelFormatter(null)
+        setOnTouchListener { v: View, event: MotionEvent ->
+            val action = event.action
+            when (action) {
+                MotionEvent.ACTION_DOWN -> v.parent.requestDisallowInterceptTouchEvent(
+                    true
+                )
+
+                MotionEvent.ACTION_UP -> {
+                    v.parent.requestDisallowInterceptTouchEvent(false)
+                    listener?.onValueChange(this, value, true)
+                }
+            }
+            v.onTouchEvent(event)
+            true
+        }
+    }
+
+    @SuppressLint("RestrictedApi")
+    override fun onStopTrackingTouch(slider: Slider) {
+        isTrackingTouch = false
+    }
+
+    @SuppressLint("RestrictedApi")
+    override fun onStartTrackingTouch(slider: Slider) {
+        isTrackingTouch = true
+    }
+
+    fun enable() {
```

Comment:
> We've ended up with two different enabled/disabled concepts for Slider

I don't think this feedback got addressed. To expand on this, it's confusing to have `enable`/`disable` because `View` already has a `isEnabled` property. My thinking that we could have something like a single `reset` method instead that does basically what `disable` does (and is called by a widget when the answer is cleared) and that the `enable` part could be handled entirely within `TrackingTouchSlider` when a user change happens.

Reply:
Done.